modules/atom-ide-ui/pkg/atom-ide-diagnostics-ui/lib/getDiagnosticDatatip.js

```diff
@@ -31,14 +32,22 @@ function makeDatatipComponent(
 ): React.ComponentType<*> {
   const fixer = message => diagnosticUpdater.applyFix(message);
   return bindObservableAsProps(
-    observableFromSubscribeFunction(cb =>
-      diagnosticUpdater.observeCodeActionsForMessage(cb),
-    ).map(codeActionsForMessage => ({
-      messages,
-      fixer,
-      goToLocation: gotoLine,
-      codeActionsForMessage,
-    })),
+    Observable.zip(
```

Comment:
i think you want `combineLatest()` here? otherwise you're depending on getting exactly one code action to every description.